### PR TITLE
new-feature:add k3redisutil

### PR DESCRIPTION
**Pull request by IloveKanade**
The k3redisutil modified according to the specification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+pip install k3redisutil
+```
+
+# Synopsis
+
+```python
+
+```
+
```
Comment (drmingdrmer) [inline README.md]: `make readme` does not fill the synopsis here?

File: redis_proxy_cli.py [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+    # Do nothing if the key doesn’t exist.
+    #
+    # **arguments**:
+    #
+    # -   `key`:
+    # specifies the key to redis.
+    #
+    # -   `retry`:
+    # try to send request for another N times while failed to send request.
+    # By default, it is `0`.
```
Comment (drmingdrmer) [inline redis_proxy_cli.py]: Why keep these a non-doc comment? It seems like these should be document.

File: test/synopsis.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import k3redisutil
+import time
+
+c = k3redisutil.RedisChannel(6379, '/foo', 'client')
+s = k3redisutil.RedisChannel(6379, '/foo', 'server')
+
+c.send_msg('c2s')
```
Comment (drmingdrmer) [inline test/synopsis.py]: Is this file a duplicate of `/synopsis.py`?

Comment (IloveKanade) [reply test/synopsis.py]: sry!!my bad....

**Review by drmingdrmer — APPROVED** [review]
LGTM

**Pull request merged**